addon/lib/common.js

```diff
@@ -99,26 +102,14 @@ export function isComponent(targetClass) {
  * @returns {Boolean}
  */
 export function isGlimmerComponent(targetClass) {
-  const flattenedPrototypeKeys = new Set();
-
-  if (!targetClass) return false;
-
+  if (!GlimmerComponent) return false;
   let ancestor = targetClass;
-  let hasGlimmerDebugComponentAncestor = false;
-
   while (ancestor) {
-    if (ancestor.name === 'GlimmerDebugComponent') hasGlimmerDebugComponentAncestor = true;
+    if (ancestor.name === 'GlimmerDebugComponent') return true;
+    if (ancestor === GlimmerComponent) return true;
```

Comment:
It seems like this is the same fix as #11 -- I think you can remove the constructor name check entirely, as a production build will _always_ have this fail.

Reply:
@alexlafroscia That doesn't seem to be my experience when I run in prod, but I do like your solution for this better.  We can merge your work in #11 once it's ready and then I will rebase this branch on top of that.

Reply:
@alexlafroscia Ah, yes, the name will get removed in prod.  I don't even think that's needed anymore either way.  Will remove that.